projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -39,10 +39,11 @@ public function __construct( $feed ) {
 	 * @return array|WP_Error  The player data or a error object.
 	 */
 	public function get_player_data( $args = array() ) {
-		$guids = isset( $args['guids'] ) && $args['guids'] ? $args['guids'] : array();
+		$guids           = isset( $args['guids'] ) && $args['guids'] ? $args['guids'] : array();
+		$episode_options = isset( $args['episode-options'] ) && $args['episode-options'];
 
 		// Try loading data from the cache.
-		$transient_key = 'jetpack_podcast_' . md5( $this->feed . implode( ',', $guids ) );
+		$transient_key = 'jetpack_podcast_' . md5( $this->feed . implode( ',', $guids ) . "-$episode_options" );
```

Comment:
minor: `$episode_options` seems to be a boolean, however, it's treated as a string here. Should we tweak it?

Reply:
Yes, I suppose I am being a bit lazy there! The idea is to make a unique key. The boolean will become 1 or an empty string, so something like:

```php
php > $test=true;
php > echo 'test' . "-$test";
test-1
php > $test=false;
php > echo 'test' . "-$test";
test-
```

Do you think it would be better if we set it to an explicit string value?